### ISSUE TITLE
rdctl start doesn't return and doesn't change container engine

### ISSUE THREAD

**Issue by jandubois** [rancher-sandbox/rancher-desktop#2113]
Before I shut down RD I was running it with the moby runtime. Then I started it up with from the CLI:

```console
PS C:\Users\Jan\Downloads> rdctl start --container-engine containerd
About to launch C:\Users\Jan\AppData\Local/Programs/Rancher Desktop/Rancher Desktop.exe --kubernetes-container-engine containerd ...

[8380:0426/202102.226:ERROR:gpu_init.cc(446)] Passthrough is not supported, GL is disabled, ANGLE is
(node:7088) [DEP0123] DeprecationWarning: Setting the TLS ServerName to an IP address is not permitted by RFC 6066. This will be ignored in a future version.
                                     (Use `Rancher Desktop --trace-deprecation ...` to show where the warning was created)
```

It does start up RD again, but it did not change the container engine to `containerd`.

The `rdctl start` command also never returned to the command prompt. When I aborted it with Ctrl-C then RD was stopped as well (not just the Window, but also the background app).

Finally there is the issue of the noisy output, but that is secondary to the functional issues.

**Comment by jandubois** [comment]
On macOS the command returned to the command prompt, but also did not change the container engine (I was running with `moby` so I tried to switch to `containerd`):

```console
$ rdctl start --container-engine moby
About to launch /usr/bin/open -a /Applications/Rancher Desktop.app --args --kubernetes-container-engine moby ...
$
```

I haven't looked at the source, but I notice that `rdctl` uses `--container-engine`[^1], but passes it on as `--kubernetes-container-engine`; maybe that is part of the issue.

[^1]: Of course the option should really be called `--container-runtime` to match the label in the UI, but that can be changed in the refactoring before the API goes to `v1`.

**Comment by mook-as** [comment]
I believe this is fixed in #2117 and #2118.

**Comment by ericpromislow** [comment]
Yes, this was fixed in those.
